data/QuoteApproval/installer-manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "wire-plugin": [
+        {
+            "target": "\\Spryker\\Client\\QuoteRequest\\QuoteRequestDependencyProvider::getQuoteRequestQuoteCheckPlugins",
```

Comment:
The usability of FQCN class names in json is always a bit problematic
I wonder if it would be useful to have those in yml files?
Any technical issues with that?